zircon.core/src/commonMain/kotlin/org/hexworks/zircon/internal/fragment/impl/DefaultMultiSelect.kt

```diff
@@ -0,0 +1,119 @@
+package org.hexworks.zircon.internal.fragment.impl
+
+import org.hexworks.cobalt.databinding.api.createPropertyFrom
+import org.hexworks.zircon.api.Components
+import org.hexworks.zircon.api.behavior.TextHolder
+import org.hexworks.zircon.api.fragment.MultiSelect
+import org.hexworks.zircon.api.graphics.Symbols
+import org.hexworks.zircon.api.uievent.ComponentEventType
+
+class DefaultMultiSelect<T : Any>(
+        width: Int,
+        override val values: List<T>,
+        override val callback: (oldValue: T, newValue: T) -> Unit,
+        private val centeredText: Boolean = true,
+        private val toStringMethod: (T) -> String = Any::toString,
+        private val clickable: Boolean = false
```

Comment:
This is only used when initializing this object so it doesn't have to be a field.

Reply:
That's absolutely correct, but why did IntelliJ not tell me? =(